ert3/config/_parameters_config.py

```diff
@@ -205,6 +232,16 @@ def as_distribution(self) -> ert3.stats.Distribution:
                 index=index,
                 size=size,
             )
+        elif dist_config.type == "loguniform":
+            assert dist_config.input.lower_bound is not None
+            assert dist_config.input.upper_bound is not None
```

Comment:
Linking this to the comment above. And again I see that this check is done for all distribution types, so I guess there is a reason to it. But still I am curious why this is not done in the `_<distribution_name>Input` instead, instead of accepting `None`?

Reply:
These two comments might come outside of the PR review, but placing them here while I have them.

Reply:
It seems to me that these asserts do not really need to be there. I don't think those values can be `None`.

Reply:
But perhaps should be moved in a separate PR?

Reply:
The asserts may be overly cautious, but don't hurt, let's just leave them.